homework-functions/index.js

```diff
@@ -74,20 +74,20 @@ function capitalizeFirstLetter(string, toCapital = true) {
 
 function createAutoComplete(array) {
```

Comment:
This function is really hard to read. Maybe better to add some comments or split it into some more functions?